2023-03-07 16:11:56,326 INFO: Website monitoring startup [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py:92]
2023-03-07 16:12:34,175 INFO: Website monitoring startup [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py:92]
2023-03-07 16:16:09,064 INFO: Website monitoring startup [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py:92]
2023-03-07 16:16:33,350 INFO: Website monitoring startup [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py:92]
2023-03-07 16:19:49,198 INFO: Website monitoring startup [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py:92]
2023-03-07 16:19:56,232 ERROR: Exception on / [GET] [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py", line 229, in homepage
    websites = current_user.websites.all()
               ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'InstrumentedList' object has no attribute 'all'
2023-03-07 16:19:57,126 ERROR: Exception on / [GET] [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py", line 229, in homepage
    websites = current_user.websites.all()
               ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'InstrumentedList' object has no attribute 'all'
2023-03-07 16:20:03,172 ERROR: Exception on / [GET] [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py", line 229, in homepage
    websites = current_user.websites.all()
               ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'InstrumentedList' object has no attribute 'all'
2023-03-07 16:20:03,750 ERROR: Exception on / [GET] [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py", line 229, in homepage
    websites = current_user.websites.all()
               ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'InstrumentedList' object has no attribute 'all'
2023-03-07 16:20:06,729 ERROR: Exception on / [GET] [in /Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/venv/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maksimkisliak/PycharmProjects/FlaskWatchdog/app.py", line 229, in homepage
    websites = current_user.websites.all()
               ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'InstrumentedList' object has no attribute 'all'
